### ISSUE TITLE
[Bug] Series not displaying - Unsupported display set

### ISSUE THREAD

**Issue by sylvainb31** [OHIF/Viewers#4302]
### Describe the Bug

I am trying to display a study in the OHIF viewer but there is a series that doesn't display, with the message "Unsupported display set". I have tried to display the same study in an other viewer (meddream) and it works, but with a very specific display with images sorted as "Phase 1", "Phase 2" and "Phase 3".

I have checked the metadata of the series not displaying, and there is a sequence present in the first image of the series that is not present elsewhere (00082112 - Source Image Sequence). Each element of the sequence points toward one of the 3 other images in the series :

![image](https://github.com/user-attachments/assets/4f72ba56-d117-4d82-bdf0-8844d723c7a3)

Here is the display that doesn't work in OHIF :

![image](https://github.com/user-attachments/assets/ccd4a52f-88ec-4eb0-a840-a3bb4df89ffe)

Here is the same study displayed in meddream that works properly :

![image](https://github.com/user-attachments/assets/d3d1bc1a-c85e-4648-80eb-a61e80d82926)

Is it this sequence (00082112 - Source Image Sequence) that prevent OHIF from displaying the series ?

Thanks for your help,

### Steps to Reproduce

Displaying a a study with the latest version of OHIF (3.8.0)

### The current behavior

A specific series does not display with the message "Unsupported display set"

### The expected behavior

For the series to display.

### OS

Windows 11

### Node version

20.14.0

### Browser

Chrome 126.0.6478.128

**Comment by sylvainb31** [comment]
I have tried loading only the list of metadata from this url : https://github.com/OHIF/Viewers/issues/3093

Images are loaded from DCM4CHEE.

PatientID => value present
PatientName => value present
PatientSex => value present
PatientAge => value present
PatientBirthDate => value present
PatientSize **=> value not present**
PatientWeight **=> value not present**
StudyInstanceUID => value present
SeriesInstanceUID => value present
SOPInstanceUID => value present
SOPClassUID => value present
Modality => value present
NumberOfFrames **=> value not present**
Rows  => value present
Columns  => value present
SamplesPerPixel  => value present
PhotometricInterpretation  => value present
PlanarConfiguration **=> value not present**
BitsAllocated  => value present
BitsStored  => value present
HighBit  => value present
PixelRepresentation  => value present
PixelAspectRatio **=> value not present**
InstanceNumber  => value present
SeriesDate  => value present
SeriesTime  => value present
FrameTime **=> value not present**
ImagePositionPatient **=> value not present**
ImageOrientationPatient	 **=> value not present**
ImageType **=> value not present**
PixelSpacing **=> value not present**
StudyDescription **=> value not present**
SeriesNumber  => value present
SeriesDescription **=> value not present**

With this list of dicom tags for each instance for the metadata, the DX series still displays correctly, and I have still the same result for the CR series, so it doesn't come from the dicom tag (00082112 - Source Image Sequence).

Is it because of the missing values in one of the dicom tags that the CR series doesn't display and show the message "Unsupported display set" ?

Thanks for your help.

**Comment by arunvc** [comment]
Try this sample, works fine in other viewers but fails to open in ohif 
[1.3.46.670589.11.44665.5.0.7624.2024090512181649196.zip](https://github.com/user-attachments/files/16909841/1.3.46.670589.11.44665.5.0.7624.2024090512181649196.zip)

**Comment by arunvc** [comment]
![image](https://github.com/user-attachments/assets/de79556e-c675-4ebc-a74a-850ffe14a385)

first entry empty and 0 series is breaking

**Comment by sylvainb31** [comment]
Hello,
I am writing you again for the same problem. Is there anything I can do to fix it and make the study display ?

![image](https://github.com/user-attachments/assets/2fa6c15e-7c3f-4b44-a621-298864cac2b6)

Once again, this study display correctly with meddream (and also using a simple cornerstone viewer), but the images don't display in the ohif viewer once the metadata query is done and read. Is there something I can do in the ohif configuration so that it displays it ?

Thanks for your help.

**Comment by sedghi** [comment]
Can you check latest OHIF 3.9?

**Comment by sedghi** [comment]
viewer.ohif.orgg

**Comment by martinopiniator** [comment]
> Try this sample, works fine in other viewers but fails to open in ohif [1.3.46.670589.11.44665.5.0.7624.2024090512181649196.zip](https://github.com/user-attachments/files/16909841/1.3.46.670589.11.44665.5.0.7624.2024090512181649196.zip)

this on shows the thumbs now

![image](https://github.com/user-attachments/assets/356f5421-f2ba-4699-9db5-3591d824d86c)

**Comment by cristianmiranda** [comment]
@sedghi I don't think this is resolved though. @martinopiniator shows that the "displaySet not found" error keeps occurring.
I've tested with multiple releases (3.8, 3.9, 3.10 and latest [v3.10.0-beta.5](https://github.com/OHIF/Viewers/releases/tag/v3.10.0-beta.5)). I get the following (tried different studies and they all work in other viewers). Is this a configuration thing? Should I somehow register Hanging Protocols since they're not being loaded by default?. Sorry I might be asking silly questions, I looked everywhere and couldn't find something related to this.

Thanks in advance!

![image](https://github.com/user-attachments/assets/00af402c-8550-4f71-846c-39b7bd5ee69d)
